Java/src/main/java/Planes/Plane.java

```diff
@@ -23,12 +23,12 @@ public int getMS() {
         return maxSpeed;
     }
 
-    public int Get_Max_Flight_Distance() {
+    public int getMaxFlightDistance() {
         return maxFlightDistance;
     }
 
     public int getMinLoadCapacity() {
-        int result = this.maxLoadCapacity;
+        int result = maxLoadCapacity;
```

Comment:
check if the "result" variable is necessary , if not, then return the value directly